Lib/modulefinder.py

```diff
@@ -17,6 +17,7 @@
 _PKG_DIRECTORY = 5
 _C_BUILTIN = 6
 _PY_FROZEN = 7
+_NAMESPACE = 8
```

Comment:
This change feels wrong based on the comment above ("Old imp constants"), because it introduces a "new old constant". Perhaps the comment just needs to be replaced with something more accurate.

Reply:
I share the same concern, but these constants are being used here to identify the module type, so maybe we can change the comment to "constants initially imported from imp module" or something similar.